src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -60,6 +62,14 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
 
   constructor(private readonly context: vscode.ExtensionContext) {}
 
+  private updateWebview(document: vscode.TextDocument, webviewPanel: vscode.WebviewPanel) {
+    this._oneConfig = ini.parse(document.getText());
+    webviewPanel.webview.postMessage({
+      type: 'displayCfgToEditor',
+      text: ini.parse(document.getText()),
```

Comment:
Can we reuse `_oneConfig`?

```suggestion
      text: this._oneConfig,
```

Reply:
I missed this, thanks :D